content/03-reference/01-tools-and-interfaces/04-introspection.mdx

```diff
@@ -35,15 +35,41 @@ You can learn more about how Prisma maps types from the database to the types av
 
 You can introspect your database using the `prisma introspect` command of the [Prisma CLI](prisma-cli/installation). Note that using this command requires your [connection URL](../database-connectors/connection-urls) to be set in your Prisma schema!
 
-> **Warning**: The `prisma introspect` command overwrites the current version of your Prisma schema! If you made any manual adjustments to the Prisma schema (e.g. by adding [comments](prisma-schema/prisma-schema-file), [making a 1-1-relation required on both sides](prisma-schema/relations#making-both-sides-required-in-one-to-one-relations) or [configuring your Prisma Client API](prisma-client/configuring-the-prisma-client-api)), be sure to back up your schema before running the command!
-
 Here's a high-level overview of the steps that `prisma introspect` performs internally:
 
 1. Read the [connection URL](../database-connectors/connection-urls) from the `datasource` configuration in the Prisma schema
 1. Open database connection
 1. Introspect database schema (i.e. read tables, columns and other structures ...)
 1. Transform database schema into Prisma data model
-1. Write data model into Prisma schema
+1. Write data model into Prisma schema or [update existing schema](#re-introspection)
+
+## Re-introspection
```

Comment:
I am not sure "Re-Introspection" should be a special concept we maintain. We used it as a technical term to describe this functionality, but now it really could just be "Introspection" and how it works. "Introspection with an existing schema" this section could be called if we want to keep it - but I see no real need to keep this term around.

Reply:
Yes, I think so too.